static/app/components/onboardingWizard/sidebar.tsx

```diff
@@ -142,15 +157,30 @@ class OnboardingWizardSidebar extends Component<Props> {
 
   render() {
     const {collapsed, orientation, onClose} = this.props;
-    const {all, active, upcoming, complete} = this.segmentedTasks;
+    const {all, active, upcoming, complete, customized} = this.segmentedTasks;
 
     const completeList = (
       <CompleteList key="complete-group">
         <AnimatePresence initial={false}>{complete.map(this.renderItem)}</AnimatePresence>
       </CompleteList>
     );
+    const customizedCards = customized
+      .map(
+        task =>
+          task.Component &&
+          task.Component({
```

Comment:
👎 to rendering react components like this.

This is dangerous because _hooks will not work_ in what appears to be a component.

Reply:
This is the only way, because if we render it as a component, animate presence & layout animation will not work, since animate presence requires that the component to be a direct child. And we also can't do `
      customizedCards.length > 0 && customizedTasksHeading`

Reply:
In that case do not call this `task.Component`, it should be `task.renderTask` or something like that.

Reply:
ok